src/rubicon/objc/api.py

```diff
@@ -815,7 +794,9 @@ def objc_class(self):
     def _associated_attr_key_for_name(name):
         return SEL(f"rubicon.objc.py_attr.{name}")
 
-    def __new__(cls, object_ptr, _name=None, _bases=None, _ns=None):
+    def __new__(
+        cls, object_ptr, _name=None, _bases=None, _ns=None, returned_from_method=b""
+    ):
```

Comment:
The docstring paragraph starting "Rubicon currently does not perform any automatic memory management" should be updated.

Reply:
Done.